scripts/i18n_utility/README.md

```diff
@@ -0,0 +1,106 @@
+# I18n Utility Script
+
+This is a CLI utility for:
```

Comment:
I'd change "CLI" to "command-line" here

I'd also change "for" to "to" so that you can so you can use the infinitive (for example, to: identify, to: delete, to: identify, etc)